### ISSUE TITLE
Syslog bystander warning. TopLevel Constant

### ISSUE THREAD

**Issue by kwent** [markrebec/blabbermouth#5]
Hi,

Having this message when starting the app.

``` plain
/usr/local/opt/rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/blabbermouth-syslog-0.0.13/lib/blabbermouth/syslog/bystander.rb:55: warning: toplevel constant Logger referenced by Blabbermouth::Bystanders::Syslog::Logger
```

Regards,

**Comment by markrebec** [comment]
Taking a look at this, will get it patched up and released. Thanks for the report!

**Comment by markrebec** [comment]
This has been fixed in the `0.1.0` release.

**Comment by kwent** [comment]
:+1:
